modules/graph/src/graph/sbom/mod.rs

```diff
@@ -60,22 +60,11 @@ impl Graph {
 
         let SearchOptions { sort, q } = search;
 
-        let mut select =
-            sbom::Entity::find().filter(Filter::<sbom::Entity>::from_str(&q)?.into_condition());
-
-        if !sort.is_empty() {
-            for s in sort
-                .split(',')
-                .map(Sort::<sbom::Entity>::from_str)
-                .collect::<Result<Vec<_>, _>>()?
-                .iter()
-            {
-                select = select.order_by(s.field, s.order.clone());
-            }
-        }
-        select = select.order_by_desc(sbom::Column::Id);
-
-        let limiter = select.limiting(&connection, paginated.offset, paginated.limit);
+        let limiter = sbom::Entity::find().filtering(&q, &sort)?.limiting(
```

Comment:
Useful to just take the `SearchOptions` instead of a pair of Strings?

Reply:
Good call! Done!